plugins/repository-s3/src/test/java/org/opensearch/repositories/s3/S3BlobContainerRetriesTests.java

```diff
@@ -118,6 +120,7 @@
  * This class tests how a {@link S3BlobContainer} and its underlying AWS S3 client are retrying requests when reading or writing blobs.
  */
 @SuppressForbidden(reason = "use a http server")
+@ThreadLeakScope(ThreadLeakScope.Scope.NONE)
```

Comment:
why do we need this change ?

Reply:
Test case were failing intermittently with few threads still active. Looks to be flaky tests.

Reply:
Lets handle it in separate CR .

Reply:
Done.. Have added a new ThreadFilter `EventLoopThreadFilter`